poker_ai/ai/multiprocess/server.py

```diff
@@ -68,7 +68,7 @@ def __init__(
         self._worker_status: Dict[str, str] = dict()
         self._agent: Agent = Agent(agent_path)
         self._locks: Dict[str, mp.synchronize.Lock] = dict(
-            regret=mp.Lock(), strategy=mp.Lock()
+            regret=mp.Lock(), strategy=mp.Lock(), pre_flop_strategy=mp.Lock()
```

Comment:
Please check me on this, seems to work as expected.

Reply:
all good